src/data/seq/seq.lean

```diff
@@ -676,6 +707,15 @@ end
 theorem mem_append_left {s₁ s₂ : seq α} {a : α} (h : a ∈ s₁) : a ∈ append s₁ s₂ :=
 by apply mem_rec_on h; intros; simp [*]
 
+@[simp] lemma enum_cons (s : seq α) (x : α) :
```

Comment:
I'm moving `set.ext` much earlier in the file in #15830. I presume doing this will allow you to move this proof earlier.